idea/testData/intentions/convertPropertyGetterToInitializer/expectedProperty.kt

```diff
@@ -0,0 +1,11 @@
+// ENABLE_MULTIPLATFORM
+// ERROR: Expected declaration must not have a body
+// IS_APPLICABLE: false
+expect class C {
+    val p: Int
+        <caret>get() = 1
```

Comment:
For multi-platform testing, we have a specific group of tests. In your case, `multiModuleQuickFix` group should work properly (it should work for intentions too). This should also solve your problem with target platform detection.